<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Strip</class>
 <widget class="QDialog" name="Strip">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>850</width>
    <height>120</height>
   </rect>
  </property>
  <property name="sizePolicy">
   <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
    <horstretch>0</horstretch>
    <verstretch>0</verstretch>
   </sizepolicy>
  </property>
  <property name="minimumSize">
   <size>
    <width>850</width>
    <height>120</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>850</width>
    <height>120</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <widget class="QTextEdit" name="strip_1">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>150</width>
     <height>60</height>
    </rect>
   </property>
   <property name="sizePolicy">
    <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
     <horstretch>0</horstretch>
     <verstretch>0</verstretch>
    </sizepolicy>
   </property>
  </widget>
  <widget class="QTextEdit" name="strip_3">
   <property name="geometry">
    <rect>
     <x>150</x>
     <y>0</y>
     <width>160</width>
     <height>40</height>
    </rect>
   </property>
   <property name="sizePolicy">
    <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
     <horstretch>0</horstretch>
     <verstretch>0</verstretch>
    </sizepolicy>
   </property>
  </widget>
  <widget class="QTextEdit" name="strip_2">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>60</y>
     <width>150</width>
     <height>60</height>
    </rect>
   </property>
   <property name="sizePolicy">
    <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
     <horstretch>0</horstretch>
     <verstretch>0</verstretch>
    </sizepolicy>
   </property>
  </widget>
  <widget class="QTextEdit" name="strip_4">
   <property name="geometry">
    <rect>
     <x>150</x>
     <y>40</y>
     <width>160</width>
     <height>80</height>
    </rect>
   </property>
   <property name="sizePolicy">
    <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
     <horstretch>0</horstretch>
     <verstretch>0</verstretch>
    </sizepolicy>
   </property>
  </widget>
  <widget class="QTextEdit" name="strip_5">
   <property name="geometry">
    <rect>
     <x>310</x>
     <y>0</y>
     <width>120</width>
     <height>60</height>
    </rect>
   </property>
   <property name="sizePolicy">
    <sizepolicy hsizetype="Fixed" vsizetype="Minimum">
     <horstretch>0</horstretch>
     <verstretch>0</verstretch>
    </sizepolicy>
   </property>
  </widget>
  <widget class="QTextEdit" name="strip_6">
   <property name="geometry">
    <rect>
     <x>310</x>
     <y>60</y>
     <width>120</width>
     <height>60</height>
    </rect>
   </property>
  </widget>
  <widget class="QTextEdit" name="strip_7">
   <property name="geometry">
    <rect>
     <x>430</x>
     <y>0</y>
     <width>200</width>
     <height>80</height>
    </rect>
   </property>
   <property name="sizePolicy">
    <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
     <horstretch>0</horstretch>
     <verstretch>0</verstretch>
    </sizepolicy>
   </property>
   <property name="font">
    <font>
     <pointsize>18</pointsize>
     <weight>75</weight>
     <bold>true</bold>
     <underline>true</underline>
    </font>
   </property>
  </widget>
  <widget class="QTextEdit" name="strip_9">
   <property name="geometry">
    <rect>
     <x>630</x>
     <y>0</y>
     <width>160</width>
     <height>120</height>
    </rect>
   </property>
   <property name="sizePolicy">
    <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
     <horstretch>0</horstretch>
     <verstretch>0</verstretch>
    </sizepolicy>
   </property>
  </widget>
  <widget class="QTextEdit" name="strip_8">
   <property name="geometry">
    <rect>
     <x>430</x>
     <y>80</y>
     <width>260</width>
     <height>40</height>
    </rect>
   </property>
   <property name="sizePolicy">
    <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
     <horstretch>0</horstretch>
     <verstretch>0</verstretch>
    </sizepolicy>
   </property>
  </widget>
  <widget class="QTextEdit" name="strip_10">
   <property name="geometry">
    <rect>
     <x>790</x>
     <y>0</y>
     <width>60</width>
     <height>40</height>
    </rect>
   </property>
   <property name="sizePolicy">
    <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
     <horstretch>0</horstretch>
     <verstretch>0</verstretch>
    </sizepolicy>
   </property>
  </widget>
  <widget class="QTextEdit" name="strip_11">
   <property name="geometry">
    <rect>
     <x>790</x>
     <y>40</y>
     <width>60</width>
     <height>40</height>
    </rect>
   </property>
  </widget>
  <widget class="QTextEdit" name="strip_12">
   <property name="geometry">
    <rect>
     <x>790</x>
     <y>80</y>
     <width>60</width>
     <height>40</height>
    </rect>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
